incubator/xray/values.yaml

```diff
@@ -0,0 +1,163 @@
+# Default values for Xray HA.
+# This is a YAML-formatted file.
+# Beware when changing values here. You should know what you are doing!
+# Access the values with {{ .Values.key.subkey }}
+
+# Common
+replicaCount: 1
+imagePullPolicy: Always
+initContainerImage: "alpine:3.6"
+imagePullSecrets:
+
+# PostgreSQL
+## Configuration values for the postgresql dependency
+## ref: https://github.com/kubernetes/charts/blob/master/stable/postgressql/README.md
+##
+postgresql:
+  enabled: true
+  postgresDatabase: "xraydb"
+  postgresUser: "xray"
+  postgresPassword: "xray"
+  service:
+    port: 5432
+  persistence:
+    enabled: true
+    size: 50Gi
+    existingClaim:
+  resources:
+    requests:
+      memory: "1Gi"
+      cpu: "250m"
+    limits:
+      memory: "2Gi"
+      cpu: "1"
+
+## Configuration values for the mongodb dependency
+## ref: https://github.com/kubernetes/charts/blob/master/stable/mongodb/README.md
+##
+mongodb:
+  enabled: true
+  image:
+    tag: 3.6.3
+    pullPolicy: Always
+  persistence:
+    size: 100Gi
+  resources:
+    requests:
+      memory: "12Gi"
+      cpu: "200m"
+    limits:
+      memory: "12Gi"
+      cpu: "2"
+  ## Make sure the --wiredTigerCacheSizeGB is no more than half the memory limit!
+  ## This is critical to protect against OOMKill by Kubernetes!
+  mongodbExtraFlags:
+  - "--wiredTigerCacheSizeGB=6"
+  mongodbRootPassword: password
+  mongodbUsername: xray
+  mongodbPassword: password
+  mongodbDatabase: xray
+#  livenessProbe:
+#    initialDelaySeconds: 60
+#    periodSeconds: 10
+#  readinessProbe:
+#    initialDelaySeconds: 30
+#    periodSeconds: 30
+
+# RabbitMQ HA
+## Configuration values for the rabbitmq-ha dependency
+## ref: https://github.com/kubernetes/charts/blob/master/stable/rabbitmq-ha/README.md
+##
+rabbitmq-ha:
+  replicaCount: 1
+  rabbitmqUsername: "guest"
+  rabbitmqPassword: "guest"
+  customConfigMap: true
+  rabbitmqErlangCookie: XRAYRABBITMQCLUSTER
+  rabbitmqMemoryHighWatermark: 500MB
+  resources:
+    requests:
+      memory: "250Mi"
+      cpu: "100m"
+    limits:
+      memory: "550Mi"
+      cpu: "200m"
+  persistentVolume:
+    enabled: true
+    size: 20Gi
+  rbac:
+    create: false
+
+common:
+  xrayVersion: 2.1.0
+  xrayConfigPath: /var/opt/jfrog/xray/data
+  xrayUserId: 1035
+  xrayGroupId: 1035
+  stdOutEnabled: true
+  stagingMode:
+    enabled: false
+    storeUser:
+    storePassword:
+  ## Xray requires a unique master key
+  ## You can generate one with the command:
+  ## 'openssl rand -hex 32'
+  ## Pass it to helm with '--set common.masterKey=${MASTER_KEY}'
+  ## IMPORTANT: You should NOT use the example masterKey for a production deployment!
+  masterKey: FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+  storage:
+    sizeLimit: 10Gi
+  resources:
```

Comment:
Don't specify defaults for resources. Comment them out. Also, you should probably make them configurable per container.